index.js

```diff
@@ -1943,6 +1943,29 @@
         return either.isRight ? [either.value] : [];
       }));
 
+  //# toEither :: a -> b? -> Either a b
+  //.
+  //. Takes a failure value, 'x', and a possibly null or undefined value, 'y'.
+  //. Returns a Right of 'y' if it is defined and not null, otherwise returns
+  //. a Left of 'x'.
+  //.
+  //. See also ['encaseEither'](#encaseEither)
+  //.
+  //. ```javascript
+  //. > S.toEither("No string", "ftp://example.com/")
+  //. Right("ftp://example.com/")
+  //.
+  //. > S.toEither("Invalid protocol", "ftp://example.com/".match(/^https?:\/\//))
+  //. Left("Invalid protocol")
+  //. ```
```

Comment:
I like the symmetry of the examples in #83. It shows that `s.match(/^(http:|https:)[/][/]/)` can result in `null` or some other value depending on the value of `s`.

I think the examples should either be realistic, as in #83, or very simple (e.g. `S.toEither('XXX', null)` and `S.toEither('XXX', 42)`).